jenkins/test-history/gen_html.py

```diff
@@ -57,6 +57,28 @@
 ])
 
 
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+
+JINJA_ENV = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(BASE_DIR + '/templates/'),
+    extensions=['jinja2.ext.autoescape'],
+    trim_blocks=True,
+    autoescape=True)
+
+
+def failure_class(passed, failed):
+    if failed == 0:
+        return ''
+    elif passed == 0:
+        return ' job-broken'
```

Comment:
oh, this is actually unnecessary. jinja is smart enough to trim the attribute for me!